dbschema/partnership.esdl

```diff
@@ -0,0 +1,34 @@
+module default {
+  type Partnership extending Project::Child {
+    agreementStatus: Partnership::AgreementStatus;
+    mou: File; 
+    mouStatus: Partnership::AgreementStatus;
+    mouStart := .mouStartOverride ?? .project.mouStart;
+    mouEnd := .mouEndOverride ?? .project.mouEnd;
+    mouStartOverride: cal::local_date;
+    mouEndOverride: cal::local_date;
+    agreement: File;
+    required partner: Partner;
+    required organization: Organization;
+    multi types: Partner::Type;
+    financialReportingType: Partnership::FinancialReportingType;
+    primary: bool {
+      default := false;
+    }
+  }
+}
+
```

Comment:
If you could actually keep the indentation on these empty lines or just add some spacing on these root level ones, it would help me out. My syntax highlighting breaks without them. I know it's dumb. I don't expect it to be this way forever.

Reply:
not sure if the changes I made help, but just added more space.